apps/server/src/modules/user/service/user.service.ts

```diff
@@ -34,6 +33,13 @@ export class UserService {
 		this.logger.setContext(UserService.name);
 	}
 
+	async getUserEntityWithRoles(userId: EntityId): Promise<User> {
+		// only roles required, no need for the other populates
```

Comment:
Den Kommentar versteh ich nicht. In der Methode spielt Populierung ja gar keine Rolle. Was soll der mir sagen?

Reply:
Doch leider das true ist das populate. Es gibt keine spezifische repo Methode. Allerdings wird viel mehr populiert als notwendig ist. Ich wollte nur nicht auch noch eine neue Repo Methode einfügen.

Reply:
Ahja okay. Aber aus der Perspektive von jemand, der irgendwann mal diesen Code liest, weiß ich trotzdem nicht so recht, was ich mit dem Kommentar anfangen soll.